### PR TITLE
Deprecate PHP 5

### PR DESCRIPTION
**Pull request by florianpreusner**
- [x] Bug fix
- [ ] New feature
- [ ] BC breaks
- [ ] Deprecations
- [x] Tests pass

Fixed tickets: #96
License: MIT

### CODE REVIEW

**Comment by gregurco** [comment]
@florianpreusner Do you need help with this PR? I have experience with php 7.0/7.1 and can help with some modifications.

**Comment by florianpreusner** [comment]
@gregurco Sure, what love to get some help here! Thanks!

File: .travis.yml [hunk 1]
```diff
@@ -1,16 +1,11 @@
 language: php
 
 php:
-    - 5.6
-    - 7.0
```
Comment (ruudk) [inline .travis.yml]: You removed the 7.0 tests... but composer.json says php >= 7.0

Comment (florianpreusner) [reply .travis.yml]: good shot!

**Comment by florianpreusner** [comment]
@ruudk What exactly do you mean?

**Comment by ruudk** [comment]
I mean, what about just removing PHP 7.0 support? PHP 7.2 will be released next month.

**Comment by gregurco** [comment]
@florianpreusner @ruudk https://seld.be/notes/php-versions-stats-2017-1-edition see this article. There are 36% of users, who use php 7.0 now...

**Comment by ruudk** [comment]
So? They can use the older version right?

**Comment by florianpreusner** [comment]
Sure, but do we have any disadvantages by supporting 7.0?

**Comment by gregurco** [comment]
The main features I use:
- nullable types (parameters and return) `function(?int $a) { }`
- void methods `function() : void { }`

**Comment by florianpreusner** [comment]
Will be continued in PR https://github.com/8p/GuzzleBundle/pull/127

**Pull request closed without merging**